.circleci/config.yml

```diff
@@ -124,7 +124,7 @@ commands:
                     if [[ $line == *"The local node will continue running at"* ]]; then
                         break
                     fi
-                    if [[ $line == *"An error occured during build"* ]]; then
+                    if [[ $line == *"An error occurred during build"* ]]; then
```

Comment:
I know it was misspelled, but this is exactly how it is written in cannon output and we must match. Ofc if you know for sure it was corrected recently, then sure we need to follow and update it. Is this the case?

Reply:
Oh okay, I thought it was just a typo, I'll revert it